src/vs/workbench/contrib/url/browser/trustedDomains.ts

```diff
@@ -208,6 +208,8 @@ export function readStaticTrustedDomains(accessor: ServicesAccessor): IStaticTru
 	const environmentService = accessor.get(IBrowserWorkbenchEnvironmentService);
 
 	const defaultTrustedDomains = [
+		// MEMBRANE: add membrane.io to the list of trusted domains to skip confirmation dialog
+		'https://membrane.io',
```

Comment:
This isn't necessary anymore with the new approach to log out without opening a new tab, but I think we can leave it in. Small UX improvement.